packages/tools/stylelint-no-logical-props-shorthands/README.md

```diff
@@ -0,0 +1,56 @@
+# stylelint-config-pie
+
+[![npm version](https://badge.fury.io/js/@justeattakeaway%stylelint-no-logical-props-shorthands.svg)](https://badge.fury.io/js/@justeattakeaway%stylelint-no-logical-props-shorthands)
+
+A Stylelint plugin used in PIE – Just Eat Takeaway’s design system.
+
+This plugin was created to prevent the usage of logical property shorthands in CSS. This is because they are not fully supported in Safari versions 14 and 15, and we want to avoid using them in our design system.
```

Comment:
I believe Safari 15 supports them?

Reply:
That issue was also found with Safari 15.6. Please have a look at DSW-1111 😉